frontend/src/context/leftPanelStateSlice.ts

```diff
@@ -1,8 +1,14 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import type { RootState } from './store';
 
+export enum Panel {
+  Layers = 0,
```

Comment:
we could even use explicit string here no? or are the numbers useful?
Layers = "layers", ...